.github/workflows/test_migrations.yaml

```diff
@@ -3,16 +3,21 @@ name: Test Skyportal Migrations
 on:
   # Run only if potential changes in database schema
   pull_request:
-    paths:
```

Comment:
@acrellin One question, I guess, is whether this should go back in?

Reply:
Ah, yeah probably good to restrict the runs to changes to these. Shall I add that back in a new commit?

Reply:
Thanks, that'd be great; perhaps also set the reference commit to the point where the new data loader reporting got merged in, then we'll have the advantage of that when running migration tests.